packages/web3-eth/test/integration/defaults.test.ts

```diff
@@ -0,0 +1,525 @@
+
+			// after set
+			web3Eth.setConfig({
+				handleRevert: true,
+			});
+			expect(web3Eth.handleRevert).toBe(true);
+
+			// set by create new instance
+			const eth2 = new Web3Eth({
+				provider: clientUrl,
+				config: {
+					handleRevert: true,
+				},
+			});
+			expect(eth2.handleRevert).toBe(true);
+		});
+		it('defaultBlock', async () => {
+			// default
+			expect(web3Eth.defaultBlock).toBe('latest');
+
+			// after set
+			web3Eth.setConfig({
+				defaultBlock: 'earliest',
+			});
+			expect(web3Eth.defaultBlock).toBe('earliest');
+
+			// set by create new instance
+			const eth2 = new Web3Eth({
+				provider: clientUrl,
+				config: {
+					defaultBlock: 'earliest',
+				},
+			});
+			expect(eth2.defaultBlock).toBe('earliest');
+			const acc = await createNewAccount({ refill: true });
+			// check implementation
+			const balance = await eth2.getBalance(acc.address);
+			expect(balance).toBe('0x0');
+			eth2.setConfig({
+				defaultBlock: 'latest',
+			});
+			const balanceLatest = await eth2.getBalance(acc.address);
+			expect(Number(hexToNumber(balanceLatest))).toBeGreaterThan(0);
+		});
+		it('transactionBlockTimeout', () => {
+			// default
+			expect(web3Eth.transactionBlockTimeout).toBe(50);
+
+			// after set
+			web3Eth.setConfig({
+				transactionBlockTimeout: 1,
+			});
+			expect(web3Eth.transactionBlockTimeout).toBe(1);
+
+			// set by create new instance
+			const eth2 = new Web3Eth({
+				provider: clientUrl,
+				config: {
+					transactionBlockTimeout: 120,
+				},
+			});
+			expect(eth2.transactionBlockTimeout).toBe(120);
+		});
+		it('transactionConfirmationBlocks', () => {
+			// default
+			expect(web3Eth.transactionConfirmationBlocks).toBe(24);
+
+			// after set
+			web3Eth.setConfig({
+				transactionConfirmationBlocks: 3,
+			});
+			expect(web3Eth.transactionConfirmationBlocks).toBe(3);
+
+			// set by create new instance
+			const eth2 = new Web3Eth({
+				provider: clientUrl,
+				config: {
+					transactionConfirmationBlocks: 4,
+				},
+			});
+			expect(eth2.transactionConfirmationBlocks).toBe(4);
+			// implementation tested here ./watch_transaction.test.ts
+		});
+		it('transactionPollingInterval and transactionPollingTimeout', () => {
+			// default
+			expect(web3Eth.transactionPollingInterval).toBe(1000);
+			expect(web3Eth.transactionPollingTimeout).toBe(750);
+
+			// after set
+			web3Eth.setConfig({
+				transactionPollingInterval: 3,
+				transactionPollingTimeout: 10,
+			});
+			expect(web3Eth.transactionPollingInterval).toBe(3);
+			expect(web3Eth.transactionPollingTimeout).toBe(10);
+
+			// set by create new instance
+			const eth2 = new Web3Eth({
+				provider: clientUrl,
+				config: {
+					transactionPollingInterval: 400,
+					transactionPollingTimeout: 10,
+				},
+			});
+			expect(eth2.transactionPollingInterval).toBe(400);
+			expect(eth2.transactionPollingTimeout).toBe(10);
+		});
+		it('transactionReceiptPollingInterval and transactionConfirmationPollingInterval', () => {
```

Comment:
transactionReceiptPollingInterval and transactionConfirmationPollingInterval seems alias for transactionPollingInterval and transactionPollingTimeout and are not in 1.x

Reply:
yep. we used both like this:
`web3Context.transactionReceiptPollingInterval ?? web3Context.transactionPollingInterval`